common/network_constants.cc

```diff
@@ -1,23 +1,45 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 #include "brave/common/network_constants.h"
 
-const char kBraveUpdatesExtensionsEndpoint[] = "https://go-updater.brave.com/extensions";
+// const char kBraveUpdatesExtensionsEndpoint[] =
+//    "https://go-updater.brave.com/extensions";
 // For debgugging:
-// const char kBraveUpdatesExtensionsEndpoint[] = "http://localhost:8192/extensions";
+const char kBraveUpdatesExtensionsEndpoint[] =
+    "http://18.191.178.76/extensions";
```

Comment:
To-Do: Revert to `localhost:8192` once the dev changes for go-updater and signed-off by the QA team.

Reply:
NOTE: this is pointing at the dev version of Go Updater

Reply:
let's wait until this is set to the real URL before merging (since HTTP is insecure). and then https://github.com/brave/brave-browser/pull/3452/files#diff-82b17b6b5b773946564b2b58b637ff29R32 can also be removed.